packages/beacon-node/src/network/peers/peerManager.ts

```diff
@@ -620,6 +620,11 @@ export class PeerManager {
     metrics.peerLongLivedAttnets.reset();
     metrics.peerConnectionLength.reset();
 
+    // reset client counts _for each client_ to 0
+    for (const client of Object.values(ClientKind)) {
+      peersByClient.set(client, 0);
```

Comment:
we default to 0 at the line 631/636 below already, do you mean `metrics.peersByClient.set({client}, 0);`?

Reply:
If all connections for a client gets pruned, no entry for that client is added to the map, so metrics never get set to 0

Reply:
oh I see it now 👍